### DISCUSSION TITLE
Parbad Virtual Gateway Language

### DISCUSSION THREAD

**Discussion by 7sepehr** [Sina-Soltani/Parbad#231] [Q&A]
امکان تغییر عبارات در درگاه ویرچوال و فارسی کردن آن وجوددارد
آیا می توان در config
عبارتهایی مثل pay 
را به پرداخت تبدیل کرد؟

**Comment by Sina-Soltani** [comment]
در حال حاضر خیر چنین امکانی موجود نیست
